app/src/sharedTest/java/org/oppia/android/app/databinding/DrawableBindingAdaptersTest.kt

```diff
@@ -0,0 +1,235 @@
+      TestDispatcherModule::class, ApplicationModule::class,
+      LoggerModule::class, ContinueModule::class, FractionInputModule::class,
+      ItemSelectionInputModule::class, MultipleChoiceInputModule::class,
+      NumberWithUnitsRuleModule::class, NumericInputRuleModule::class, TextInputRuleModule::class,
+      DragDropSortInputModule::class, ImageClickInputModule::class, InteractionsModule::class,
+      GcsResourceModule::class, TestImageLoaderModule::class, ImageParsingModule::class,
+      HtmlParserEntityTypeModule::class, QuestionModule::class, TestLogReportingModule::class,
+      AccessibilityTestModule::class, LogStorageModule::class, CachingTestModule::class,
+      PrimeTopicAssetsControllerModule::class, ExpirationMetaDataRetrieverModule::class,
+      ViewBindingShimModule::class, RatioInputModule::class, WorkManagerConfigurationModule::class,
+      ApplicationStartupListenerModule::class, LogUploadWorkerModule::class,
+      HintsAndSolutionConfigModule::class, HintsAndSolutionProdModule::class,
+      FirebaseLogUploaderModule::class, FakeOppiaClockModule::class, PracticeTabModule::class,
+      DeveloperOptionsStarterModule::class, DeveloperOptionsModule::class,
+      ExplorationStorageModule::class, NetworkModule::class, NetworkConfigProdModule::class,
+      NetworkConnectionUtilDebugModule::class, NetworkConnectionDebugUtilModule::class,
+      AssetModule::class, LocaleProdModule::class, ActivityRecreatorTestModule::class
+    ]
+  )
+  /** Create a TestApplicationComponent. */
+  interface TestApplicationComponent : ApplicationComponent {
+    /** Build the TestApplicationComponent. */
+    @Component.Builder
+    interface Builder : ApplicationComponent.Builder
+
+    /** Inject [DrawableBindingAdaptersTest] in TestApplicationComponent . */
+    fun inject(drawableBindingAdaptersTest: DrawableBindingAdaptersTest)
+  }
+
+  /**
+   * Class to override a dependency throughout the test application, instead of overriding the
+   * dependencies in every test class, we can just do it once by extending the Application class.
+   */
+  class TestApplication : Application(), ActivityComponentFactory, ApplicationInjectorProvider {
+    private val component: TestApplicationComponent by lazy {
+      DaggerDrawableBindingAdaptersTest_TestApplicationComponent.builder()
+        .setApplication(this)
+        .build() as TestApplicationComponent
+    }
+
+    /** Inject [DrawableBindingAdaptersTest] in TestApplicationComponent . */
+    fun inject(drawableBindingAdaptersTest: DrawableBindingAdaptersTest) {
+      component.inject(drawableBindingAdaptersTest)
+    }
+
+    override fun createActivityComponent(activity: AppCompatActivity): ActivityComponent {
+      return component.getActivityComponentBuilderProvider().get().setActivity(activity).build()
+    }
+
+    override fun getApplicationInjector(): ApplicationInjector = component
+  }
+
+  private interface Consumer<T> {
```

Comment:
I suspect this also isn't needed.

Reply:
Thanks, @BenHenning  as mentioned, this [function](https://github.com/oppia/oppia-android/pull/4077#discussion_r785239269) is not being used, which makes it redundant.